src/components/InlineInput/InlineInput.tsx

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import React from 'react';
+
+import { NumberFormatValues, NumericFormat } from 'react-number-format';
+
+interface InlineInputProps {
+  name: string;
+  disabled: boolean;
```

Comment:
I would say `disabled` is not mandatory. 🤔 

```suggestion
  disabled?: boolean;
```